shacl12-test-suite/tests/core/node/memberShape-001.ttl

```diff
@@ -0,0 +1,102 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://datashapes.org/sh/tests/core/node/memberShape-001.test#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:IRIListShape
+  rdf:type sh:NodeShape ;
+  sh:memberShape [ sh:nodeKind sh:IRI ] ;
+  sh:targetNode ( ex:Alice ex:Bob ) ;
+  sh:targetNode () ;
+  sh:targetNode _:b1, _:b2, _:b3, _:b4, _:b5 ;
```

Comment:
`_:b3` isn't among the data.

Reply:
That is intended. It should raise a violation because a blank node with no properties is an invalid list.